packages/editor/src/store/effects/reusable-blocks.js

```diff
@@ -184,7 +184,10 @@ export const deleteReusableBlocks = async ( action, store ) => {
 	] ) );
 
 	try {
-		await apiFetch( { path: `/wp/v2/${ postType.rest_base }/${ id }`, method: 'DELETE' } );
+		await apiFetch( {
+			path: `/wp/v2/${ postType.rest_base }/${ id }?force=true`,
```

Comment:
Thinking about this some more, I think we should remove the `?force=true` so that deleting a reusable block in Gutenberg matches what happens when you delete a reusable block in the `Manage Reusable Blocks` interface.

Reply:
I'm okay with this. As alluded to, seems like it would make sense to have this be controlled at the post-type level, for the sort of consistency you note. In the meantime, letting "Trashed" do its thing seems quite okay with me.

Reply:
Related to this one, we'll need to do similar with the forced "Published" status.